examples/wind-polar-selections.py

```diff
@@ -0,0 +1,96 @@
+from dash import Dash, html, dcc, callback, Input, Output
+import dash_bootstrap_components as dbc
+import plotly.express as px
+
+app = Dash(__name__, external_stylesheets=[dbc.themes.CYBORG])
```

Comment:
This app looks great with the Cyborg theme, but in production it will default to the main app's theme.  It still looks OK.  No need to change anything here.

Reply:
Noted - thanks